app/controllers/coronavirus_form/additional_product_controller.rb

```diff
@@ -18,7 +18,7 @@ def submit
     if invalid_fields.any?
       flash.now[:validation] = invalid_fields
       render "coronavirus_form/#{PAGE}"
-    elsif additional_product == I18n.t("coronavirus_form.questions.additional_product.options.option_yes.label")
+    elsif additional_product == YES
```

Comment:
Was this just a nice tidy whilst you were here, or does this help with fragment caching too?

Reply:
Actually I'm not sure what the impact of using a constant is on performance. It's just a refactor to make things clearer :)

Reply:
cool, very nice!